organizations/models.py

```diff
@@ -29,10 +40,18 @@ def _generate_unique_schema_name(self):
             schema_name = self._generate_random_string()
         return schema_name
 
+    def _generate_unique_api_key(self):
+        """Generates a unique api_key."""
+        api_key = self._generate_random_cryptographically_secure_string()
+        while Organization.objects.filter(api_key=api_key).exists():
+            api_key = self._generate_random_cryptographically_secure_string()
```

Comment:
this line is not covered in tests.

Reply:
There is no way to test this
<img width="655" alt="Screenshot 2021-06-30 at 1 01 34 PM" src="https://user-images.githubusercontent.com/20169829/123920189-4e362500-d9a3-11eb-98cf-1758e75a9e6f.png">